__tests__/signup.test.ts

```diff
@@ -15,6 +15,58 @@ interface User {
 }
 
 describe('createUser', () => {
```

Comment:
I would make the describe title more descriptive. Something like "testing error path createUser...". So when running the tests we see what is tested in the describe block.